### PR TITLE
BAH-2229 | Added translation messages for photo upload

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**

### CODE REVIEW

File: ui/app/common/photo-capture/directives/capturePhoto.js [hunk 1]
```diff
@@ -138,11 +138,11 @@ angular.module('bahmni.common.photoCapture')
                     var fileType = this.files[0].type;
                     if (!fileType.startsWith('image/')) {
                         uploadConfirmImageButton.prop('disabled', true);
-                        alert('File uploaded must be an image');
+                        alert($translate.instant("FILE_UPLOAD_MUST_BE_IMAGE"));
```
Comment (gsluthra) [inline ui/app/common/photo-capture/directives/capturePhoto.js:141]: These are existing keys in translation files?

Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js:141]: No added new ones.

Comment (gsluthra) [reply ui/app/common/photo-capture/directives/capturePhoto.js:141]: Should the translation file also be in this PR then?

Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js:141]: I have added it in clinic-config repo and raised PR there.

Comment (SooryaKumaranC-tw) [reply ui/app/common/photo-capture/directives/capturePhoto.js:141]: > Should the translation file also be in this PR then?

Added translation file in bahmniapps too.

**Review by gsluthra — APPROVED** [review]

**Pull request merged**